winui/Masked-TextBox/How-To.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Validation support in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the validation support in the MaskedTextBox (SfMaskedTextBox) control.
```

Comment:
@BhaskarSuresh22 Learn all about the restrict the validation support in the MaskedTextBox (SfMaskedTextBox) control.

Reply:
@Manivannan-E, Done the changes.